src/main/java/hudson/plugins/ws_cleanup/PreBuildCleanup.java

```diff
@@ -27,18 +32,22 @@
 
 	private static final Logger LOGGER = Logger.getLogger(PreBuildCleanup.class.getName());
 
-	private final List<Pattern> patterns;
-	private final boolean deleteDirs;
-	private final String cleanupParameter;
-	private final String externalDelete;
-	private final boolean disableDeferredWipeout;
+	private List<Pattern> patterns = Collections.emptyList();
+	private boolean deleteDirs;
+	private String cleanupParameter = StringUtils.EMPTY;
+	private String externalDelete = StringUtils.EMPTY;
+	private boolean disableDeferredWipeout;
+
+	@DataBoundConstructor
+	public PreBuildCleanup() {
+	}
 
 	@Deprecated
 	public PreBuildCleanup(List<Pattern> patterns, boolean deleteDirs, String cleanupParameter, String externalDelete) {
 		this(patterns, deleteDirs, cleanupParameter, externalDelete, false);
 	}
 
-	@DataBoundConstructor
+	@Deprecated
```

Comment:
Now that this constructor is deprecated, all callers in test code should be migrated to a non-deprecated equivalent. I suggest using an IDE's Call Hierarchy feature to find and update these callers.